internal/cmd/api-key/command_create.go

```diff
@@ -133,6 +134,21 @@ func (c *command) create(cmd *cobra.Command, _ []string) error {
 		}
 	}
 
+	use, err := cmd.Flags().GetBool("use")
+	if err != nil {
+		return err
+	}
+	if use {
```

Comment:
Can you write an integration test for this, or break it out into a unit test?

Reply:
added an int test